crud/select/executor.lua

```diff
@@ -11,13 +11,12 @@ local ExecuteSelectError = errors.new_class('ExecuteSelectError')
 
 local executor = {}
 
-local function scroll_to_after_tuple(gen, param, state, space, scanner)
-    local scan_index = space.index[scanner.index_id]
+local function scroll_to_after_tuple(gen, param, state, space, scan_index, iter, after_tuple)
```

Comment:
You don't need all of this options.

`gen, param, state` could be reduced to `gen`

```lua
tarantool> s = box.space.log:pairs()
---
...

tarantool> s(s.param, s.state)
---
- <iterator state>
- tuple
...
```

Seven arguments... It's too big.